dat/events/scavenger_event.lua

```diff
@@ -0,0 +1,2077 @@
+        _(fmt.f("Tip Pilot {credits}", {credits = fmt.credits(tip_amount)})),
+        _("Go scavange"),
+        _("Call back"),
+        _("Do nothing")
+    )
+    if
+        n == 1 and
+            tk.yesno(
+                "",
+                fmt.f(
+                    _("Are you sure you want to tip {name}? This will cost {price}."),
+                    {name = edata.name, price = fmt.credits(tip_amount)}
+                )
+            )
+     then
+        --	  mem.persons[index].total_profit = mem.persons[index].total_profit - tip_amount -- do the credit side now since debit comes later
+        shiplog.append(
+            logidstr,
+            fmt.f(
+                _("You tipped '{name}' {credits} ({ship})."),
+                {name = edata.name, ship = edata.ship, credits = fmt.credits(tip_amount)}
+            )
+        )
+        player.pilot():credits(-tip_amount)
+        edata.wallet = edata.wallet + tip_amount
+        if edata.pilot ~= nil and edata.pilot:exists() then
+            edata.pilot:credits(tip_amount)
+        end
+        mem.persons[index].total_cost = mem.persons[index].total_cost + tip_amount
+    elseif n == 2 then
+        edata.pilot:setLeader(nil)
+        --for i, follower in ipairs(player.pilot():followers()) do
+        for i, persona in ipairs(mem.persons) do
+            local follower = persona.pilot
+            if follower and rnd.rnd() > 0.5 and follower ~= edata.pilot and follower:exists() then
+                speak(persona, "join", edata)
+                follower:setLeader(edata.pilot)
+            end
+        end
+    elseif n == 3 then
+        edata.pilot:setLeader(player.pilot())
+    end
+end
+
+function scav_hail(p, arg)
+    -- Remove randomness from future calls
+    if not mem.hailsetup then
+        mem.refuel_base = mem.refuel_base or rnd.rnd(2000, 4000)
+        mem.bribe_base = mem.bribe_base or math.sqrt(p:stats().mass) * (300 * rnd.rnd() + 850)
+        mem.bribe_rng = rnd.rnd()
+        mem.hailsetup = true
+    end
+
+    -- Clean up
+    mem.refuel = 0
+    mem.refuel_msg = nil
+    mem.bribe = 0
+    mem.bribe_prompt = nil
+    mem.bribe_prompt_nearby = nil
+    mem.bribe_paid = nil
+    mem.bribe_no = nil
+
+    -- Deal with refueling
+    local standing = p:faction():playerStanding()
+    mem.refuel = mem.refuel_base
+    if standing > 60 then
+        mem.refuel = mem.refuel * 0.5
+    end
+    mem.refuel_msg = fmt.f(_([["For you, only {credits} for a jump of fuel."]]), {credits = fmt.credits(mem.refuel)})
+
+    -- Deal with bribeability
+    mem.bribe = mem.bribe_base
+    if mem.allowbribe or (mem.natural and mem.bribe_rng < 0.95) then
+        mem.bribe_prompt = fmt.f(bribe_prompt_list[rnd.rnd(1, #bribe_prompt_list)], {credits = fmt.credits(mem.bribe)})
+        mem.bribe_prompt_nearby = bribe_prompt_nearby_list[rnd.rnd(1, #bribe_prompt_nearby_list)]
+        mem.bribe_paid = bribe_paid_list[rnd.rnd(1, #bribe_paid_list)]
```

Comment:
`bribe_prompt_list`, `bribe_prompt_nearby`, and `bribe_paid_list` is not actually defined anywhere...